docs/platforms/javascript/guides/react/features/component-tracking.mdx

```diff
@@ -4,6 +4,84 @@ excerpt: ""
+
+There will be more features released down the line that will leverage component name tracking, so it is highly recommended to configure your project to use it.
+
+### How to Install
+
+<Alert>
+
+Please note that your Sentry browser SDK must be at version `7.91.0` or higher before you can use these features.
+Only React components in `.jsx` or `.tsx` files can be tracked.
+
+</Alert>
+
+There are two different ways you can setup React component name tracking in your application:
+
+1. Install [Sentry's bundler plugins](https://www.npmjs.com/package/@sentry/bundler-plugin-core)
+2. Directly install [@sentry/babel-plugin-component-annotate](https://www.npmjs.com/package/@sentry/babel-plugin-component-annotate)
+
+It's recommended that you use the first option, as the Sentry bundler plugins come packed with additional useful features to enrich your Sentry workflows, such as automatic sourcemap upload,
+release creation, automatic release name discovery, and release injection. However, you have the option of installing the standalone Babel plugin as well.
+
+**Setup via bundler plugins**:
+
+Follow the instructions in the npm page for the bundler that your project uses:
+- [Rollup](https://www.npmjs.com/package/@sentry/rollup-plugin)
+- [Vite](https://www.npmjs.com/package/@sentry/vite-plugin)
+- [Webpack](https://www.npmjs.com/package/@sentry/webpack-plugin)
+
+Please note that although there is a Sentry bundler plugin for **esbuild**, React component name tracking is not currently supported by it.
+
+Once you've followed the instructions to install the bundler plugin and added it to your bundler's config, enable component name tracking by setting the flag
+to enable it to `true`:
+
+
+```javascript
+// Example specific to Vite, see documentation for other bundlers
+sentryVitePlugin({
+    // ... other options above
+    reactComponentAnnotation: { enabled: true }
+}),
+```
+
+<Alert>
+
+**Important Note**: If you are using **Rollup** as your bundler, ensure that you place `sentryRollupPlugin` before any other plugins that transform your JSX
+
+</Alert>
+
+Now, you will be able to see your component names in breadcrumbs, spans, and search for replays by component names.
+See [this page](/platforms/javascript/guides/react/features/component-tracking/#component-name-tracking) for more information on searching for Replays by component name.
+
```

Comment:
```suggestion
Now you're all set to be able to see your component names in breadcrumbs and spans, and to search for replays by component name.
```

Reply:
Looks good, but why remove the link here?